content/plugins/joshembling-image-optimizer.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Image Optimizer
+slug: image-optimizer
```

Comment:
Please update the slug to match the file name. Should be 'joshembling-image-optimizer'